docs/docs/data-structures/tree/schema-evolution/allowed-types-rollout.mdx

```diff
@@ -32,7 +32,7 @@ synchronizeTrees();
 
 ### Step 1: Define and Stage the New Allowed Type
 
-To add support for reading strings, we import the alpha APIs and use [`staged`](../../../api/fluid-framework/schemafactoryalpha-class#staged-property) to mark the new allowed type:
+To add support for reading strings, we import the alpha APIs and use [`staged`](../../../api/fluid-framework/schemafactorybeta-class.md) to mark the new allowed type:
```

Comment:
@Josmithr Can you confirm these are what we want. I think they are right, but you are the expert here.

Reply:
Yeah. This will avoid causing a website build break when the next release branch is cut. We can restore the heading IDs after that point.

We have work filed to find a better solution to the underlying problem here, but it hasn't been scheduled yet.